src/stan/mcmc/hmc/hamiltonians/softabs_point.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef STAN_MCMC_HMC_HAMILTONIANS_SOFTABS_POINT_HPP
+#define STAN_MCMC_HMC_HAMILTONIANS_SOFTABS_POINT_HPP
+
+#include <stan/interface_callbacks/writer/base_writer.hpp>
+#include <stan/mcmc/hmc/hamiltonians/ps_point.hpp>
+
+namespace stan {
+  namespace mcmc {
+
+    // Point in a phase space with a base
```

Comment:
(nice to have) doxygen class and function --- it's the one form of doc I fully support